fission/log/log.go

```diff
@@ -27,7 +27,7 @@ var (
 )
 
 func Fatal(msg interface{}) {
-	os.Stderr.WriteString(fmt.Sprintf("%v\n", msg))
+	os.Stderr.WriteString(fmt.Sprintf("[FATAL] %v\n", msg))
```

Comment:
These messages are not logs; they're meant for humans.  So something like `"Fatal error: ..."` is better than `"[FATAL] ..."`